app/src/interfaces/list-m2m/list-m2m.vue

```diff
@@ -1,282 +1,373 @@
 <template>
```

Comment:
## Many to Many Feedback

Reply:
- [ ] Pagination alignment (https://github.com/directus/directus/pull/12082#discussion_r844428900)

Reply:
- [ ] Same selection bug as https://github.com/directus/directus/pull/12082#discussion_r844426830

Reply:
- [ ] Nested filter doesn't filter out correct item

https://user-images.githubusercontent.com/9141017/162075799-30e8c069-f8c5-4840-979e-b481801d3795.mp4

(`Testing` should've been filtered out, as it's already selected)

Reply:
- [ ] Can't deselect nested selected item

https://user-images.githubusercontent.com/9141017/162077078-a97c4602-f433-4740-8b5f-53084bb272f0.mp4